### DISCUSSION TITLE
Custom LabelMap on TensorRT

### DISCUSSION THREAD

**Discussion by jsbowling42** [blakeblackshear/frigate#5331] [Ask A Question]
can you setup the custom label map the same with TensorRT? i know it would depend on the model and what index the objects name is for that model that is being used. i would think it would be the same process but different object names since i think the model is different for RT.

for example i keep getting cats haha:
![image](https://user-images.githubusercontent.com/35424582/216074868-cd8e4d7f-5ff4-4255-983a-1afcbe10c7c5.png)

**Comment by NickM-27** [comment]
Based on https://deploy-preview-4055--frigate-docs.netlify.app/configuration/detectors#configuration-parameters it is the exact same process, as TRT uses the same labelmap as the CPU / Coral model

**Comment by jsbowling42** [comment]
@NickM-27 i dont see where is says that TRT used the same labelmap. it might be useful to add a note in there to point that out. thanks for your help!

**Reply by NickM-27** [reply]
It's the fact that the labelmap_path isn't overridden that means it uses the same one.
